packages/lib-classifier/src/components/Classifier/components/Layout/components/CenteredLayout/CenteredLayout.js

```diff
@@ -21,40 +21,35 @@ const Relative = styled(Box)`
 const ContainerBox = styled(Box)`
   display: flex;
   flex-direction: row;
-  gap: 1.25rem;
+  gap: 20px;
```

Comment:
This gap is `'medium'` in the other layouts because they use Grommet Grid. We do not want the gap to change size if the foot font size is changed by the user.